### PR TITLE
Add paging to get-slow-argument.py

### PR DESCRIPTION
**Pull request by maaktweluit**
When pull requests got too big buildbot stopped building them.

Changed it so it reads the next pages of the review comments too.

### CODE REVIEW

File: scripts/get-slow-argument.py [hunk 1]
```diff
@@ -13,10 +13,30 @@
 # config vars
 required_approvals = 1
 
+_logging = False
```
Comment (Krigpl) [inline scripts/get-slow-argument.py]: When is this meant to be `True`?

Comment (maaktweluit) [reply scripts/get-slow-argument.py]: when it doesnt work and you need to see what is going on

Comment (Krigpl) [reply scripts/get-slow-argument.py]: maybe read the value from an env variable instead so you don't have to modify the code

**Review by Krigpl — APPROVED** [review]
vader_didnt_read

File: scripts/get-slow-argument.py (hunk 1 above)
Comment (Krigpl) [reply scripts/get-slow-argument.py]: Or even better
- don't introduce the new `_log` function
- set `logger.propagate = False` when env variable is set

Comment (Krigpl) [reply scripts/get-slow-argument.py]: Oh, there's no logger here. Nevermind then (unless you wanna use logger).

Comment (maaktweluit) [reply scripts/get-slow-argument.py]: Good one! added :)

**Pull request merged**